apps/server/src/modules/files-storage/uc/files-storage.uc.ts

```diff
@@ -274,16 +275,20 @@ export class FilesStorageUC {
 
 		const [fileRecords, count] = await this.fileRecordRepo.findBySchoolIdAndParentId(params.schoolId, params.parentId);
 
-		if (count === 0) {
-			return [fileRecords, count];
-		}
+		if (count > 0) {
+			const response = await this.copy(userId, fileRecords, copyFilesParams.target);
 
-		const newRecords = await this.copy(userId, fileRecords, copyFilesParams.target);
+			return response;
+		}
 
-		return newRecords;
+		throw new NotFoundException('NO_FILES_TO_COPY');
```

Comment:
No it should not throw an error in this case. It break the try any logic from copy service.